lib/manager/pip_requirements/extract.spec.ts

```diff
@@ -29,7 +29,16 @@ const requirements6 = readFileSync(
   'utf8'
 );
 
+const requirements7 = readFileSync(
+  'lib/manager/pip_requirements/__fixtures__/requirements7.txt',
+  'utf8'
+);
+
 describe('lib/manager/pip_requirements/extract', () => {
+  delete process.env.PIP_TEST_TOKEN;
```

Comment:
Move that into before each please. We should also remove it after each test to be sure it would not interfere any other tests in future.

Same for trustLevel.

Reply:
Sure thing, done. FYI, I copied this construct from [lib/datasource/npm/index.spec.ts](https://github.com/renovatebot/renovate/blob/e015875ac58308dc0415830e2ef22d901955fa33/lib/datasource/npm/index.spec.ts#L26-L29), which you might want to update in the same way.